src/app/content/components/Content.browserspec.tsx

```diff
@@ -9,8 +9,8 @@ const TEST_CASES: { [testCase: string]: (target: Page) => Promise<void> } = {
   Desktop: setDesktopViewport, Mobile: setMobileViewport,
 };
 const EXPECTED_SCROLL_TOPS: { [testCase: string]: number[] } = {
-  Desktop: [242, 90, 122, 242, 365, 668, 761, 1262, 1606],
-  Mobile: [239, 66, 96, 239, 523, 1263, 1402, 1751, 2118],
+  Desktop: [227, 90, 122, 227, 365, 668, 761, 1262, 1606],
+  Mobile: [224, 66, 96, 224, 523, 1263, 1402, 1751, 2118],
```

Comment:
why are these modified?

Reply:
I'm not sure what is causing this test to fail. Sizes of the noteish elements looks the same as in `master`.

Reply:
it looks like you removed 15px of bottom padding, i think this test is legitimately failing